jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -69,11 +76,12 @@ class LogGroupFilterTableSorter(model: ListTableModel<LogStream>) : TableRowSort
 }
 
 class LogStreamDateColumn : ColumnInfo<LogStreamEntry, String>(message("general.time")) {
-    private val renderer = ResizingTextColumnRenderer()
-    override fun valueOf(item: LogStreamEntry?): String? = TimeFormatConversion.convertEpochTimeToStringDateTime(item?.timestamp, showSeconds = true)
+    private val renderer = ResizingDateColumnRenderer(showSeconds = false)
```

Comment:
On copying the timestamp, would the value be in epoch or normal date format?

Reply:
missed a revert here, but the copied text was the locale-based date format